packages/nouns-contracts/contracts/governance/NounsDAOLogicV2.sol

```diff
@@ -91,6 +91,12 @@ contract NounsDAOLogicV2 is NounsDAOStorageV2, NounsDAOEventsV2 {
     /// @notice The maximum number of actions that can be included in a proposal
     uint256 public constant proposalMaxOperations = 10; // 10 actions
 
+    /// @notice The maximum priority fee used to cap gas refunds in `castRefundableVote`
+    uint256 public constant MAX_REFUND_PRIORITY_FEE = 20 gwei;
```

Comment:
A 20 gwei priority fee feels a bit excessive. How do you feel about reducing to 10 gwei?

Reply:
was waiting to discuss this with you :)
the reason I went with 20 rather than a lower number was because I was anchored by these two implementations:

*  https://github.com/fomo-nouns/fomo-nouns/blob/main/packages/contracts/contracts/NounSettlement.sol#L17
* https://github.com/z0r0z/zolidity/blob/main/src/utils/Refunded.sol#L18

I'm really not sure why they chose such a high number.

Curious how did you land on 10 gwei? @solimander

Reply:
The latter is closer to what we want. The former (FOMO) is incredibly high because they're trying to get into the pending block. Either way, lower stakes here because the transaction will not revert if exceeded.

Thinking through it again, even 10 is very high haha. In practice, the priority fee is unlikely to exceed 2 gwei... so maybe cap somewhere in the 2-5 range.

Reply:
updated to 2 gwei